webapp/packages/core-connections/src/DatabaseAuthModelsResource.ts

```diff
@@ -16,8 +18,21 @@ import {
 
 @injectable()
 export class DatabaseAuthModelsResource extends CachedMapResource<string, DatabaseAuthModel> {
-  constructor(private graphQLService: GraphQLService) {
+  constructor(
+    private readonly graphQLService: GraphQLService,
+    private readonly permissionsService: PermissionsService
+  ) {
     super();
+
+    this.isModelAvailable = this.isModelAvailable.bind(this);
+  }
+
+  isModelAvailable(model: DatabaseAuthModel) {
+    if (this.permissionsService.has(EAdminPermission.admin)) {
+      return true;
+    }
+
+    return !model.requiresLocalConfiguration;
   }
```

Comment:
better to split to 
`const admin = usePermission(EAdminPermission.admin)` 
...
`const availableAuthModels = applicableAuthModels.filter(model => !!model && (admin || !model.requiresLocalConfiguration));`

or modify `isModelAvailable` to asynchronous function to guarantee that permissions are loaded and updated before filtering auth models